### PR TITLE
newcomer_metrics.md: Update requirements

### PR DESCRIPTION
**Pull request by himanshu1603**
This adds a prerequisite for newcomers to solve issues from coala/community repository as these issues are suitable for them.

Fixes https://github.com/coala/projects/issues/450

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by himanshu1603** [comment]
@jayvdb Can you please review my PR.
Thanks

**Comment by aashraybhandar1** [comment]
It should be closes instead of fixes

**Comment by aashraybhandar1** [comment]
And I think the commit message should be something like that
`Adds a prerequisite for newcomers to solve issues from coala/community repository`

**Comment by himanshu1603** [comment]
I've formatted the commit message as per commit guidelines please review it.

File: _projects/newcomer_metrics.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ name: "Newcomer metrics and gamification"
 requirements:
   - "At least one patch to each of coala-landing and coala-html should be accepted and merged."
   - "The student should be familiar with web design tools & trends."
+prereq: "Add PRs to coala/community repository https://github.com/coala/community/ "
```
Comment (sks444) [inline _projects/newcomer_metrics.md]: @himanshu1603 , you don't have to create another section for `prereq`, basically this issue  means that you have to replace the first requirement of the project e.g: `At least one patch to each of coala-landing and coala-html should be accepted and merged.` will be something like `The student should have patches in coala/community repo`

**Comment by sks444** [comment]
@himanshu1603 , please squash your commits into one :)

File: _projects/newcomer_metrics.md [hunk 2]
```diff
@@ -11,9 +11,8 @@ mentors:
   - jayvdb
 name: "Newcomer metrics and gamification"
 requirements:
-  - "At least one patch to each of coala-landing and coala-html should be accepted and merged."
+  - "The student should have patches in coala/community repo."
```
Comment (sks444) [inline _projects/newcomer_metrics.md]: @himanshu1603 , `patches` and `PRs` seems more appropriate :)

**Comment by himanshu1603** [comment]
Please review these changes

**Comment by himanshu1603** [comment]
Please review changes

**Comment by sks444** [comment]
@himanshu1603 , No worries ; don't create new PR. follow these steps:
1. run `git log`
2. copy the head of the commit which is just above the author of the last update to the repo. 
3. run `git reset --soft <head>`
4. now do `git status` and see the file you want push
5. run `git add <file name>`
6. run `git commit` and type your commit message
7. run `git push origin <this branch name> --force`
see [this](http://api.coala.io/en/latest/Developers/Git_Basics.html#follow-up) and this [https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html](url) for more help :)

**Comment by himanshu1603** [comment]
Is it correct?

**Comment by sks444** [comment]
No, you're still having the changes from others PR and you should be only having one commit message.
@himanshu1603 , did you followed the steps? :)

**Comment by himanshu1603** [comment]
Does it look good?

**Comment by sks444** [comment]
@himanshu1603 , not quite, you have to remove that other commit :)

**Comment by himanshu1603** [comment]
will git revert <commit-id of other commit> work in this case?  Or is there any other option?

**Comment by sks444** [comment]
@himanshu1603 , try it :+1:  I am not sure.  Or have you tried rebasing yet? because the other changes you have in your PR is already merged.

**Comment by himanshu1603** [comment]
git revert is not working here.
I think now I have to use 
"git rebase master
git checkout master
git merge my_branch_name"
or just
"git rebase master" will do the work.

**Comment by sks444** [comment]
@himanshu1603 , yeah, first checkout to master and then update it with upstream if you haven't already. [see](https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository) and then checkout to your local branch and do `git rebase master` and if it says you have unstashed changes stash it by `git stash` and then do the rebase and then do `git stash pop`

**Comment by himanshu1603** [comment]
I think it should be correct now. Please check it.

**Comment by himanshu1603** [comment]
Thanks a lot for helping me so much!

**Comment by sks444** [comment]
@himanshu1603 , I am glad, I helped you :)  you tried hard! ;)

**Comment by hemangsk** [comment]
ack ae9024e

**Comment by hemangsk** [comment]
@gitmate-bot ff

**Comment by hemangsk** [comment]
@gitmate-bot rebase

**Comment by hemangsk** [comment]
ack d6c5a07

**Comment by hemangsk** [comment]
buck up travis ! ⌛️

**Pull request merged**